components/quickbooks/actions/update-customer/update-customer.mjs

```diff
@@ -1,380 +1,441 @@
-// legacy_hash_id: a_0Mi7vm
-import { axios } from "@pipedream/platform";
+import quickbooks from "../../quickbooks.app.mjs";
+import { ConfigurationError } from "@pipedream/platform";
 
 export default {
   key: "quickbooks-update-customer",
   name: "Update Customer",
   description: "Updates a customer.",
-  version: "0.1.1",
+  version: "0.1.2",
   type: "action",
   props: {
-    quickbooks: {
-      type: "app",
-      app: "quickbooks",
-    },
-    display_name: {
+    quickbooks,
+    displayName: {
```

Comment:
Can we move `displayName` to the app file? It is used here and in `create-customer`